frontend/utility/CrashHandler.cpp

```diff
@@ -38,7 +38,7 @@ constexpr std::string_view crashSentinelPrefix = "crash_sentinel_";
 constexpr std::string_view crashUploadURL = "https://obsproject.com/logs/upload";
 
 #ifndef NDEBUG
-constexpr bool isSentinelEnabled = false;
+constexpr bool isSentinelEnabled = true;
```

Comment:
Happened to me a few times as well - you want to revert this for a PR of course.. ;)